src/ReactiveUI.Maui/RoutedViewHost.cs

```diff
@@ -239,4 +183,87 @@ protected IObservable<Page> PageForViewModel(IRoutableViewModel? vm)
 
         return Observable.Return(pg);
     }
-}
\ No newline at end of file
+
+    /// <summary>
+    /// Page for view model.
+    /// </summary>
+    /// <param name="vm">The vm.</param>
+    /// <returns>An observable of the page associated to a <see cref="IRoutableViewModel"/>.</returns>
+    [SuppressMessage("Design", "CA1822: Can be made static", Justification = "Might be used by implementors.")]
+    protected Page PageForViewModel(IRoutableViewModel vm)
```

Comment:
If this might be used by implementors should this be virtual?

Reply:
Along with the other methods in there

Reply:
In general yes, but i'v just copied previous implementation. Mostly I can't see the reason to have method, returning `IObservable<Page>`...